src/utils/glue.h

```diff
@@ -211,6 +217,21 @@ struct funcbox : box {
     matryoshka_t args;
 };
 
+template<typename Klazz>
+struct getter : box {
```

Comment:
maybe) without the box)) but really, that's a minor thing. We could have a getter too. Just make it the way you like better and I merge it.